app/src/main/java/com/codesoom/assignment/HttpStatusCode.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment;
+
+/*
+    Written based on RFC2616 section 10
+    https://datatracker.ietf.org/doc/html/rfc2616#section-10
+*/
```

Comment:
기왕 주석을 작성한다면 JavaDoc 포맷을 맞추는 것이 좋겠죠. JavaDoc에 대해서도 조사해보시면 좋은 공부가 될 거에요.

```suggestion
/*
 * Written based on RFC2616 section 10
 * https://datatracker.ietf.org/doc/html/rfc2616#section-10
 */
```